pyaerocom/colocation_auto.py

```diff
@@ -378,6 +390,36 @@ def run(self, var_name=None, **opts):
         finally:
             self._close_log()
 
+    def instantiate_gridded_reader(self, what):
+        """
+        Create reader for model or observational gridded data.
+
+        Parameters
+        ----------
+        what : str
+            Type of reader. ("model" or "obs")
+        """
+        if what == 'model':
+            data_id = self.model_id
+            data_dir = self.model_data_dir
+        else:
+            data_id = self.obs_id
+            data_dir = self.obs_data_dir
+        reader_class = self._get_gridded_reader_class(what=what)
+        reader = reader_class(data_id=data_id, data_dir=data_dir)
+        if hasattr(reader, 'filepath'):
+            reader.filepath = self.filepath
```

Comment:
Is it ensured that `Colocator.filepath` is defined? I think by default there is no such attribute.

Reply:
No, this was fixed in another branch, but disappeared somewhere.
Will fix.